repo/packages/F/flink/0/marathon.json.mustache

```diff
@@ -3,7 +3,8 @@
   "cpus": {{app-master.cpus}},
   "mem": {{app-master.memory}},
   "env": {
-    "DCOS_SERVICE_NAME": "{{service.name}}"
+    "DCOS_SERVICE_NAME": "{{service.name}}",
+    "FLINK_LOG_DIR": "/mnt/mesos/sandbox"
```

Comment:
Should we $MESOS_SANDBOX instead here (https://mesosphere.github.io/marathon/docs/task-environment-vars.html)?